pkg/apis/logging/v1/kibana_types.go

```diff
@@ -5,18 +5,42 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// KibanaSpec defines the desired state of Kibana
+// Specification of the desired behavior of the Kibana
+//
 // +k8s:openapi-gen=true
 type KibanaSpec struct {
-	ManagementState ManagementState          `json:"managementState"`
-	Resources       *v1.ResourceRequirements `json:"resources"`
-	NodeSelector    map[string]string        `json:"nodeSelector,omitempty"`
-	Tolerations     []v1.Toleration          `json:"tolerations,omitempty"`
-	Replicas        int32                    `json:"replicas"`
-	ProxySpec       `json:"proxy,omitempty"`
+	// Indicator if the resource is 'Managed' or 'Unmanaged' by the operator
+	//
+	// +kubebuilder:validation:Enum:=Managed;Unmanaged
+	// +optional
```

Comment:
technically not optional because of the enum

Reply:
See above, resolving this one